_hackbars/belamo.md

```diff
@@ -0,0 +1,13 @@
+---
+Layout: hackbar
+Title:  Climate change is real
+Author: Raymond Belamo
+---
+
+When you get into developing, you would'nt really understand open source until you start writing codes. Then You will discover the power in the open source community. 
+
+---
+
+## People need to stop the climate change denial. It's as real as it gets. It's high time we understand that renewables are the key to securing a better planet. Climate change is real
+
+![Plant a tree](https://images.app.goo.gl/WY4KkEgEsuptyqFp9)
```

Comment:
@raymondbelamo Can you please add an image to this repo and link in the `_hackbars/belamo.md`
For more details, you can go through the details.